pages/blog/2022Q1-summary.md

```diff
@@ -0,0 +1,175 @@
+
+We never give up here at AsyncAPI, though :smiley:
+
+### OpenForce
+
+March was a month where we joined [OpenForce](https://www.asyncapi.com/blog/openforce-2022) thanks to amazing support from [Hargun Kaur](https://www.linkedin.com/in/hkaur008/). As a result, we managed to onboard a few new folks and gained important contributions in exchange. There are actually a few PRs still open, but for now, completed are:
+- [Namya LG](https://github.com/Namyalg) introduced a great improvement to our CI by creating dedicated workflows to [search for broken links](https://github.com/asyncapi/.github/issues/92) in our documentation across all repositories.
+- [Harsh Mishra](https://www.linkedin.com/in/harshcasper/) introduced a workflow in our CI to [validate docker images](https://github.com/asyncapi/server-api/issues/50) in projects where we publish them
+- [Abhijeet Jejurkar](https://github.com/abhijeetjejurkar) helped [improving Kubernetes deployment](https://github.com/asyncapi/server-api/issues/57) for our `server-api` service.
+- [Samriddhi](https://github.com/Samridhi-98) contributed [new `/bundle` endpoint](https://github.com/asyncapi/server-api/issues/55) to our `server-api` service.
+- [Ritik Rawal](https://github.com/ritik307) contributed [new `/diff` endpoint](https://github.com/asyncapi/server-api/issues/56) to our `server-api` service.
+- [Everly Precia Suresh](https://github.com/everly-gif) added a possibility to [host API reference docs](https://github.com/asyncapi/server-api/issues/43) for our `server-api` service through a dedicated endpoint.
+
+It is also super important to mention [Abir Pal](https://twitter.com/imabptweets), who helped to coordinate efforts and was the first point of contact for the community coming from OpenForce channels.
+
+### Google Season of Docs
+
+Success with OpenForce was followed by unexpected announcement that we were accepted as an organization for Google Season of Docs (GSoD) with $10k budget :muscle:
+
+<TwitterTweetEmbed
+  tweetId='1514670329246801926'
+  options={{
+    cards: 'hidden',
+    width: 200,
+    maxWidth: 400
+  }}
+/>
+
+The amazing success of [Alejandra Quetzalli](https://twitter.com/QuetzalliAle) and an excellent opportunity for AsyncAPI Initiative to improve documentation big time!
+In mid-May, we will have the names of six interns we want to hire to work on two different projects to have better docs. Alejandra leads a super challenging task to interview as many candidates as possible (around 200!). Keep your fingers crossed.
+
+Alejandra and I will be mentors for interns for the next few months. Stay tuned to watch the progress.
+
+### AsyncAPI Mentorship
+
+Even though the Google Summer of Code (GSoC) application was declined, we did not want to give up. We knew many community members joined and started contributing because they counted on AsyncAPI being part of GSoC. 
```

Comment:
```suggestion
Even though the Google Summer of Code (GSoC) 2022 application was declined, we did not want to give up. We knew many community members joined and started contributing because they counted on AsyncAPI being part of GSoC. 
```